### ISSUE TITLE
Code: Implement Hebrew/Greek gloss consistency check (T1)

### ISSUE THREAD

**Issue by CraigBuckmaster** [CraigBuckmaster/ScriptureDeepDive#336]
## Parent Epic
#327

## Context
267 Hebrew/Greek words have divergent glosses across chapters (8.4% of 3,159 tracked). Most are semantic synonyms, but some may be errors.

## Examples
```
חֶסֶד (16x): ['unfailing love', 'kindness / faithful love', 'covenant loyalty']
שָׁלוֹם (8x): ['peace / completeness', 'peace / wholeness', 'peace']
```

## Acceptance Criteria
- [ ] Add to `accuracy_verifiers.py` as T1 check
- [ ] Collect all `heb[].word` → `heb[].gloss` mappings across content
- [ ] Normalize glosses (lowercase, standardize separators)
- [ ] Flag words with >3 distinct normalized glosses
- [ ] Info level only — divergence is often contextually valid

## Implementation Notes
```python
word_glosses = defaultdict(set)
for word_entry in heb_panel:
    word = word_entry['word']
    gloss = normalize(word_entry['gloss'])
    word_glosses[word].add(gloss)

for word, glosses in word_glosses.items():
    if len(glosses) > 3:
        info(f"Gloss divergence: {word} has {len(glosses)} variants")
```

## Priority
P3 — nice to have, low signal-to-noise ratio

## Assignee
Claude Code
